3.0.0.Dockerfile

```diff
@@ -23,6 +23,7 @@ RUN ${WD}/common/buildbro zeek ${VER}
 # get geoip data
 
 FROM debian:stretch as geogetter
+ARG MAXMIND_LICENSE_KEY
 RUN apt-get update && apt-get -y install wget ca-certificates --no-install-recommends
 ADD ./common/getmmdb.sh /usr/local/bin/getmmdb.sh
 RUN /usr/local/bin/getmmdb.sh
```

Comment:
I think should be /usr/local/bin/getmmdb.sh ${MAXMIND_LICENSE_KEY} to pass the key as the first argument.  or need

    ENV MAXMIND_LICENSE_KEY=${MAXMIND_LICENSE_KEY}

otherwise the arg doesnt automatically become an env var

Reply:
I tested the current commit and it worked even though the ARG was not propagated via an ENV variable.

Having said that, I do prefer the model of passing the license key via command line and then testing it, so I've implemented that in the next commit :)